src/sentry/integrations/gitlab/integration.py

```diff
@@ -298,6 +320,16 @@ def build_integration(self, state):
             },
         }
 
+        # TODO(lb): do I need success to be logged?
+        logger.info(
```

Comment:
Sure, if you're going to plot it or track. Generally users don't write in when things are working though.